src/app/trips/trips-list/trips-list.component.html

```diff
@@ -25,32 +25,29 @@ <h1 class="page-title">Your Trips</h1>
 
         <!-- create new trip thumbnail -->
         <div class="col-md-6 wrapper">
-          <new-trip-thumbnail
-            [title]="createUpcomingTrip_Title"
-            [description]="createUpcomingTrip_Description">
+          <new-trip-thumbnail [title]="createUpcomingTrip_Title" [description]="createUpcomingTrip_Description">
           </new-trip-thumbnail>
         </div>
       </div>
     </div>
   </div>
 
-  <hr/>
+  <hr />
```

Comment:
This is another one of those elements to try and avoid. While this isn't as bad as br or strong, it's still an element with its only purpose as styling. You can achieve this with borders and padding. Sometimes it's ok to use an hr if it truly makes sense.